Assets/Scripts/IInteractive.cs

```diff
@@ -6,7 +6,6 @@
 /// Interface for all the things the player can interact with.
 /// </summary>
 
-
 public interface IInteractive
 {
     string DisplayText { get; }
```

Comment:
No sure if you want extra space here, but I don't think this is repeated in other files. Change as you will please.